engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/StartPlayingScreen.java

```diff
@@ -16,18 +16,101 @@
 package org.terasology.rendering.nui.layers.mainMenu;
 
 import org.terasology.assets.ResourceUrn;
+import org.terasology.config.Config;
+import org.terasology.context.Context;
+import org.terasology.engine.GameEngine;
+import org.terasology.engine.SimpleUri;
+import org.terasology.engine.TerasologyConstants;
+import org.terasology.engine.modes.StateLoading;
+import org.terasology.engine.module.ModuleManager;
+import org.terasology.game.GameManifest;
+import org.terasology.i18n.TranslationSystem;
+import org.terasology.module.DependencyResolver;
+import org.terasology.module.Module;
+import org.terasology.module.ResolutionResult;
+import org.terasology.network.NetworkMode;
+import org.terasology.registry.In;
+import org.terasology.rendering.assets.texture.Texture;
 import org.terasology.rendering.nui.CoreScreenLayer;
 import org.terasology.rendering.nui.WidgetUtil;
+import org.terasology.rendering.nui.widgets.UIImage;
+import org.terasology.rendering.nui.widgets.UILabel;
+import org.terasology.rendering.world.WorldSetupWrapper;
+import org.terasology.world.internal.WorldInfo;
+import org.terasology.world.time.WorldTime;
 
 public class StartPlayingScreen extends CoreScreenLayer {
 
+    @In
+    private ModuleManager moduleManager;
+
+    @In
+    private Config config;
+
+    @In
+    private GameEngine gameEngine;
+
+    @In
+    private TranslationSystem translationSystem;
+
     public static final ResourceUrn ASSET_URI = new ResourceUrn("engine:startPlayingScreen");
+    private Texture texture;
```

Comment:
Again, let's try to maintain with the codebase at large. Almost everywhere we start with public static final variables at top, followed by public and private variables. So let's move ASSET_URI to the top.

Also, not a huge fan of "ASSET_URI" - it does not really tell me what the asset is. See if you can come up with a better name for it.

Reply:
Every screen's urn in the game has been given the name "ASSET_URI". So I used the same on all my screens, for keeping it uniform. :-)